### ISSUE TITLE
Run I/O PnP tests with devicemapper

### ISSUE THREAD

**Issue by amshinde** [clearcontainers/runtime#349]

**Comment by egernst** [comment]
We want to do a run of this before making our relase to make sure #'s look correct, but adding this to CI shouldn't block release.

**Comment by egernst** [comment]
BTW, I assume this issue should really be "run i/o pnp CI on devicemapper", right?

**Comment by grahamwhaley** [comment]
If this is for the CI, then it is still valid. If it really was just a one off request, then it is done and can be closed.

**Comment by egernst** [comment]
Okay - I think it should be done for PnP CI, right?  Who's an appropriate owner?

**Comment by egernst** [comment]
**assuming this CI is getting migrated also for kata?

**Comment by amshinde** [comment]
Yes, this needs to be done as part of PNP CI. I have changed the title to make it clearer.

**Comment by grahamwhaley** [comment]
and the owner will be me. It may be a little tricky depending on how much hardware we have - I need to investigate if we can get away with multiple graph/mount types without having to re-config and reboot docker, but I suspect we cannot, which is where the 'tricky little dance' might come in.

**Comment by amshinde** [comment]
@grahamwhaley Yes, to switch to a different storage driver, you will need to reconfigure docker, stop all containers and basically get rid of /var/lib/docker/ before restarting docker.

**Comment by grahamwhaley** [comment]
yeah. ouch :-( Not a fun thing to try and orchestrate reliably in a CI - so, we'll see what we can do at some point.
